CONTRIBUTING.md

```diff
@@ -73,16 +73,17 @@ git push origin main
 
 Please use the below markdown template, replace the fields in ALL_CAPS with your own text.
 
+<!-- prettier-ignore-start -->
 ```md
 ---
-
 > ### 📚 **[PAPER_TITLE](PAPER_URL)**
 >
 > - 📅 **Publication Date:** MONTH DD, YYYY
 > - 📘 **Journal:** JOURNAL_NAME
 > - 🔗 **DOI:** [DOI_NUMBER](DOI_URL)
 >
-> [🔍 EG_MORE_DETAILS](URL_1)<br> > [💻 EG_LINK_2_CODE](URL_2)<br>
+> [🔍 EG_MORE_DETAILS](URL_1)<br>
+> [💻 EG_LINK_2_CODE](URL_2)<br>
```

Comment:
This is virtually the same as 
```md
> [🔍 EG_MORE_DETAILS](URL_1)
>
> [💻 EG_LINK_2_CODE](URL_2)
```
But not using `<br>` reduces the risk of messing things up with copy-pasting.

Reply:
Seams to be different in the preview and actual application, so just ignore it!

Reply:
Looks different on my machine ... and github. ^^

For lack of a better option, I'll go with this for now. Till I figure out what works best for users of the template. :)